### PR TITLE
Technical Documentation PR# Code-Review. 

### PR DESCRIPTION
**Pull request by nati2323**
-Created a new empty repo.
 -Copied my code to the new repo.
 -Created a new branch.
 -Opened a pull request

### CODE REVIEW

**Review by nati2323 — COMMENTED** [review]
-Hi! Natnael and Mushtab! Good job! almost There! But, some changes required!
download
 -for best practice of coding I think it is better if you use Span tag for paragraph.
 -I recommend if u delete unused tags like code.
 -A 'viewport' meta element was not specified. I think it would be better to use html5 standard.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @nati2323, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Good polite comments

# Required Changes ♻️

- [ ] Kindly make sure to offer an explanation of why the changes are being requested.
- [ ] Kindly add inline comments to your review, it's a good practice as it easily pinpoints the area you ar requesting the developer to make changes.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
```
Comment (nati2323) [inline index.html:4]: - A 'viewport' meta element was not specified. I think it would be better to use html5 standard.

**Review by nati2323 — COMMENTED** [review]
Hi! @Natnael and @Mushtab!

 Good job! almost There! But, still some changes required!

# Required Changes ♻️
For best practice of coding I think it is better if you use Span tag for paragraph. and I recommend if u delete un-used tags like code and it's nice if u check all indentations are correct. 

Check the comments under the review.

# Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

**Review by m05e5 — APPROVED** [review]
# Hi @nati2323 , 

### Status: Approved ✔️
goodjob



Your project is complete! 
Congratulations! 🎉 
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**